packetbeat/decoder/decoder.go

```diff
@@ -157,13 +157,11 @@ func (d *Decoder) AddLayers(layers []gopacket.DecodingLayer) {
 }
 
 func (d *Decoder) Do(jobChan <-chan *model.PacketWrapper) {
-	//logp.Info("decoder started")
-	debugf("decoder started")
+	logp.Info("decoder started")
 	for pw := range jobChan {
 		d.decode(pw.Data, pw.CaptureInfo)
 	}
-	debugf("decoder stopped")
-	// logp.Info("decoder stopped")
+	logp.Info("decoder stopped")
```

Comment:
Change from debug to info might reduce performance significantly due to disk i/o operations. I see the info "started" and "stopped" doesn't give you any useful information for debugging/troubleshooting, hence I suggest just leave it as debug. Or even remove it.